ocs_ci/utility/powernodes.py

```diff
@@ -90,6 +122,34 @@ def stop_powernodes_machines(
             if not ret.wait_for_func_status(result=True):
                 raise UnexpectedBehaviour("Node {node.name} is still Running")
 
+    def start_powernodes_machines_powervs(
+        self, powernode_machines, timeout=900, wait=True, force=True
+    ):
+        """
+        Start PowerNode Machines
+
+        Args:
+            powernode_machines (list): List of PowerNode machines
+            timeout (int): time in seconds to wait for node to reach 'not ready' state,
+                and 'ready' state.
+            wait (bool): Wait for PowerNodes to start - for future use
+            force (bool): True for PowerNode ungraceful power off, False for
+                graceful PowerNode shutdown - for future use
+        """
+        for node in powernode_machines:
+            result = exec_cmd(
+                f"ssh core@{node.name} sudo systemctl start kubelet.service"
```

Comment:
I think that node.name is not reliable value for IP address as this is not always IP address but can be also the regular name.   Node IP can be taken from status:
```
status:
  addresses:
  - address: 10.243.128.6
    type: InternalIP
  - address: 10.243.128.6
    type: ExternalIP
  - address: 10.243.128.6
    type: Hostname
```
This is other part which should be moved to service.py and you will just import it and call it here.

So I would go with ExternalIP type.

We have this function for that purpose:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/node.py#L251

So I think you should use this and get one of external IP to be used here. So maybe you can write new function in node.py which will be called get_node_ip which will return only one reachable IP.

Reply:
@petr-balogh I used InternalIP and submitted an amendment to PR. Let me know if there are any concerns.